FieldOpt/Optimization/objective/NPV.cpp

```diff
@@ -0,0 +1,123 @@
+//
+// Created by bragesk on 11.09.18.
+//
+
+#include "NPV.h"
+
+#include <iostream>
+#include <iomanip>
+#include "weightedsum.h"
+#include <stdlib.h>
+#include <cmath>
+
+using std::cout;
+using std::endl;
+using std::fixed;
+using std::setprecision;
+using std::scientific;
+
+namespace Optimization {
+    namespace Objective {
+
+        NPV::NPV(Settings::Optimizer *settings,
+                 Simulation::Results::Results *results) {
+            settings_ = settings;
+            results_ = results;
+            components_ = new QList<NPV::Component *>();
+            auto report_time = results->GetValueVector(results->Time);
+            for (int i = 0; i < settings->objective().NPV_sum.size(); ++i) {
+                auto *comp = new NPV::Component();
+                comp->property_name = settings->objective().NPV_sum.at(i).property;
+                comp->property = results_->GetPropertyKeyFromString(comp->property_name);
+                comp->coefficient = settings->objective().NPV_sum.at(i).coefficient;
+                if(settings->objective().NPV_sum.at(i).usediscountfactor == true){
+                    comp->interval = settings->objective().NPV_sum.at(i).interval;
+                    comp->discount = settings->objective().NPV_sum.at(i).discount;
+                    cout << comp->discount << endl;
```

Comment:
Remove temporary debug prints.
For prints that should be here, use the new `Printer` functions (e.g. how it's used [here](https://github.com/PetroleumCyberneticsGroup/FieldOpt/blob/develop/FieldOpt/WellIndexCalculation/wicalc_rixx.cpp#L122)